includes/blocks/class-sensei-course-outline-block.php

```diff
@@ -251,9 +251,11 @@ private function render_module_block( $block, $course_id, $outline_attributes )
 				<header class="wp-block-sensei-lms-course-outline-module__name">
 					<h2 class="wp-block-sensei-lms-course-outline__clean-heading">' . $block['title'] . '</h2>
 					' . $progress_indicator . '
-					<div role="button" tabindex="0" class="wp-block-sensei-lms-course-outline__arrow dashicons dashicons-arrow-up-alt2"/>
+					<button type="button" class="wp-block-sensei-lms-course-outline__arrow dashicons dashicons-arrow-up-alt2">
+						<span class="screen-reader-text">' . __( 'Toggle module content', 'sensei-lms' ) . '</span>
+					</button>
 				</header>
-				<div class="wp-block-sensei-lms-collapsible ' . $animated . '">
+				<div class="wp-block-sensei-lms-collapsible expanded ' . $animated . '">
```

Comment:
I think this is not needed anymore.

Reply:
Good catch! The class and the jQuery were removed here:
https://github.com/Automattic/sensei/pull/3646/commits/47b1beacb49b38a40ebe5cee919841a374efeb1e https://github.com/Automattic/sensei/pull/3646/commits/bc329629a58d02abfb23e8bbc820f797b7d93265